src/scheduler/pbs_sched_utils.cpp

```diff
@@ -776,7 +776,11 @@ wait_for_cmds()
 				if (err != 1) {
 					/* if memory error ignore, else reconnect server */
 					if (err != -2) {
+						if (sigprocmask(SIG_BLOCK, &allsigs, &oldsigs) == -1)
```

Comment:
Id suggest adding these to inside connect_svrpool and reconnect_servers, because these functions might get called from multiple places, so it’ll ensure that we don’t miss blocking the signals.

Reply:
So the problem is that in some cases reconnect_server is already called from a place where we block signals before calling it. See, schedule_wrapper()->send_cycle_end()->reconnect_server(). 
If I move it inside connect_svrpool or reconnect_server, I'll make scheduler block signals again for no particular reason. This is the reason why I am adding the change only where it is needed.

Reply:
ok, I see your point, but is there any harm in potentially blocking them again? If there's no harm then I still think that it might be safer to block them inside these functions. The reason why I brought this up is because I'm currently working on a change around these functions and I'd have probably not realized that we need to block signals before calling them from somewhere else in the code and injected this bug again.

Reply:
I've addressed your comment Ravi, please have a look.